lib/new_relic/agent/instrumentation/active_merchant.rb

```diff
@@ -36,7 +36,7 @@ class ActiveMerchant::Billing::Gateway
   end
 
   executes do
-    next unless ActiveMerchant::VERSION < Gem::Version.new('1.65.0')
+    next unless Gem::Version.new(ActiveMerchant::VERSION) < Gem::Version.new('1.65.0')
```

Comment:
👍 NOTE that this will work if `ActiveMerchant::VERSION` is a String OR an instance of `Gem::Version` already.

Reply:
This is so great. Thank you for finding this, Tanna.